### DISCUSSION TITLE
Remember last used folder when selecting view and css for blocks

### DISCUSSION THREAD

**Discussion by skttl** [umbraco/Umbraco-CMS#14768] [Features and ideas]
When adding custom views and css to blocks, it's quite a pain to navigate to the right folder for every block. It would be lovely, if it could remeber where I was last (/wwwroot/grid), and just unfold that one immediately.

![image](https://github.com/umbraco/Umbraco-CMS/assets/3726467/a6e5537d-ff30-4ac0-b032-8a419310a638)

**Comment by bjarnef** [comment]
In addition to that I think it would be great to be able to configure default path to custom views and thumnails:
https://github.com/umbraco/Umbraco-CMS/issues/9898

I think most developers have thumbnails and custom views in same folder - or regarding custom views they may exists in sub folders it they have controllers assigned, but in that case configuring the parent folder level would save some clicks.

The same could be relevant to CSS files.

Something like this:

```
<content>
   <blocks>
      <defaultCustomViewsPath>~/App_Plugins/BlockViews/</defaultCustomViewsPath>
      <defaultThumbnailsPath>~/App_Plugins/BlockViews/Thumbnails/</defaultThumbnailsPath>
   </blocks>
</content>
```

Of course JSON configuration now from Umbraco 9+
